LICENSE.rst

```diff
@@ -1,7 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2022-2024, The Trustees of Columbia University
-in the City of New York.
+Copyright (c) 2025, The Trustees of Columbia University in the City of New York.
```

Comment:
we need to keep the full range and extend it to 2025, so this would be 2022-2025 now